src/main/java/wooteco/subway/domain/line/Line.java

```diff
@@ -21,8 +30,8 @@
     private LocalDateTime createdAt;
     @LastModifiedDate
     private LocalDateTime updatedAt;
-    @Embedded.Empty
-    private LineStations stations = LineStations.empty();
+    @Embedded
+    private LineStations lineStations = LineStations.empty();
 
     public Line() {
```

Comment:
사용하는 곳이 없어 보이네요. public이 아닌 protected로 바꾸면 어떨까요?

```suggestion
    protected Line() {
```

Reply:
`LineControllerTest`에서 사용하고 있네요.